wrapanapi/systems/ec2.py

```diff
@@ -832,6 +832,36 @@ def list_templates(self, executable_by_me=True, owned_by_me=True, public=False):
         return [EC2Image(system=self, raw=self.ec2_resource.Image(image["ImageId"]))
                 for image in images]
 
+    def filter_free_images(self, image_list=None):
```

Comment:
```suggestion
    def list_free_images(self, image_list=None):
```
Lets be consistent with naming for definations!